pkg/crd/kubermatic/v1/datacenter.go

```diff
@@ -206,6 +206,10 @@ type DatacenterSpecOpenstack struct {
 	// See https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#load-balancer
 	// This setting defaults to true.
 	ManageSecurityGroups *bool `json:"manage_security_groups"`
+	// Optional: Gets mapped to the "use-octavia" setting in the cloud config.
+	// See https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/#load-balancer
+	// This setting defaults to true.
```

Comment:
nit: sorry I overlooked this, the link seems to be broken and as we discussed the setting defaults to true only when CCM is used:

https://github.com/kubernetes/cloud-provider-openstack/blob/e5e97a92de1cb69d503bf9b4891405ce5f077efd/docs/openstack-cloud-controller-manager/using-openstack-cloud-controller-manager.md#load-balancer

I would remove the link and mention that, WDYT?